src/main/java/org/gridsuite/gateway/NettyMetricsConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.gateway;
+
+import org.springframework.boot.web.embedded.netty.NettyServerCustomizer;
+import org.springframework.context.annotation.Configuration;
+import reactor.netty.http.server.HttpServer;
+
+import java.util.function.Function;
+
+// As discussed here https://stackoverflow.com/questions/66028195/spring-webflux-actuator-netty-thread-metrics,
```

Comment:
From what I read in this, I think we should add either a template for the URI or disable the metrics that we don't use.
```
	 * It is strongly recommended to provide template-like form for the URIs. Without a conversion to a template-like form,
	 * each distinct URI leads to the creation of a distinct tag, which takes a lot of memory for the metrics.
```
from their doc. 
See: https://quarkus.io/guides/telemetry-micrometer#define-tags

Reply:
you can see all the metrics it generates here: http://localhost:9000/actuator/prometheus 
if you navigate a bit in the app, it generates high cardinality metrics

Reply:
I would use something like:
https://gist.github.com/antoinebhs/8907cc2f9492ce84904d020b2d800977
to only store the metrics that we really use see documentation here: https://docs.micrometer.io/micrometer/reference/concepts/meter-filters.html
It works fine on my deployment, it only adds the two metrics (that are counter so no high cardinality issue/need to template etc)

Reply:
done